packages/tally/src/plugin-details.yml

```diff
@@ -0,0 +1,15 @@
+project:
```

Comment:
The plugin-details.ymp file needs to be moved up one directory (with the changelog and readme)